### PR TITLE
Add hint about OCP\Folder::getById

### PR DESCRIPTION
**Pull request by MorrisJobke**
@rullzer because we talked about it.

### CODE REVIEW

File: lib/public/Files/Folder.php [hunk 1]
```diff
@@ -145,6 +145,8 @@ public function searchByTag($tag, $userId);
 	/**
 	 * get a file or folder inside the folder by it's internal id
 	 *
+	 * If the file/folder is shared to you multiple times this will return more than one element
```
Comment (ChristophWurst) [inline lib/public/Files/Folder.php]: nidpick: shared *with* you

Comment (MorrisJobke) [reply lib/public/Files/Folder.php]: Fixed.

**Comment by nickvergessen** [comment]
also mention https://github.com/nextcloud/server/issues/6972 ?

**Comment by icewind1991** [comment]
Sharing is not the only way a file can exist in multiple places (external storage mainly)

File: lib/public/Files/Folder.php (hunk 1 above)
Comment (ChristophWurst) [reply lib/public/Files/Folder.php]: Actually, I was just nitpicking on the grammar: "shared to" -> "shared with" :grin:

**Comment by MorrisJobke** [comment]
> also mention #6972 ?

Let's do this in a separate PR, because it involves more than just this entry.

**Comment by MorrisJobke** [comment]
> Sharing is not the only way a file can exist in multiple places (external storage mainly)

Rephrased ;)

**Review by skjnldsv — APPROVED** [review]

**Review by icewind1991 — APPROVED** [review]

**Pull request merged**